protos/generate.sh

```diff
@@ -1,14 +1,24 @@
 #!/usr/bin/env bash
 set -e
 
-GIT_ROOT=$(git rev-parse --show-toplevel)
-cd "$GIT_ROOT"
+if [[ "$PWD" == '/' ]]; then
+  echo 'Manually set source directory to /home/bento'
+  BENTOML_SRC_DIR='/home/bento'
+else
+  BENTOML_SRC_DIR=$(git rev-parse --show-toplevel)
+fi
 
-# Install python grpcio-tools if it's not already installed
-# pip install grpcio-tools
+echo $BENTOML_SRC_DIR
 
-PROTO_PATH=$GIT_ROOT/protos
-PYOUT_PATH=$GIT_ROOT/bentoml/yatai/proto
+cd "$BENTOML_SRC_DIR"
+
+if ! [ -x "$(command -v bentoml)" ]; then
+  echo "Installing BentoML dev dependencies"
+  pip install -e .[dev]
```

Comment:
move this to the base docker image too.

Reply:
so if there's no deps change, the generate run will skip most of the installation

Reply:
also add `-U` option here, `pip install -U ..`

Reply:
actually, let's just remove this and add the following to the dockerfile:

`pip install grpcio-reflection==1.27.2 grpcio-tools==1.27.2`